docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -2,7 +2,7 @@
 title: System configuration
 headerTitle: System configuration
 linkTitle: 1. System configuration
-description: Configure NTP and ulimits on your YugabyteDB cluster
+description: How to configure system parameters correctly to get the YugabyteDB database cluster to run correctly.
```

Comment:
"correctly" is mentioned twice. We can eliminate one.

Reply:
I removed the "manually" here because these same steps can be easily automated. We don't have a page "how to automate system parameters". You do these same steps but put them in a small script.